### PR TITLE
[feature/cumulative--10.0.0] AB#33499 - Close Data Display gap between Sketch Cloud and cm-ui library (Table/Datagrid).

### PR DESCRIPTION
**Pull request by War777**
This PR includes changes for Table/Datagrid gaps under:
[PBI AB#33499](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/33499) | Close Data Display gap between Sketch Cloud and cm-ui library.

### CODE REVIEW

**Comment by groberts314** [comment]
PR Review Checkpoint Meeting Tues 2020-12-01.
Geoffrey and Cameron to review at earliest convenience

File: src/dataDisplay/dataGrid/dataGrid.jsx [hunk 1]
```diff
@@ -1,43 +1,90 @@
...
-import React, { useEffect } from 'react';
+import React from 'react';
 import { BEM_DATA_GRID } from '../../global/constants';
 import { SORTABLE_PROP_TYPES } from './dataGridConstants';
 import DataGridTable from './dataGridTable';
 import makeStyles from '../../styles/makeStyles';
 
 const propTypes = {
+    /**
+     * If <code>true</code>, the component will bleed to the edge of the parent component.
```
Comment (cbrewer1689) [inline src/dataDisplay/dataGrid/dataGrid.jsx:11]: Please just backticks instead of `<code>`.

File: src/dataDisplay/dataGrid/dataGrid.jsx [hunk 2]
```diff
@@ -1,43 +1,90 @@
...
+     * Applies small styling.
+     */
     small: PropTypes.bool,
-    stickyColumnWidth: PropTypes.number,
+    /**
+     * Enables a resisable handler in the column number set in this prop.
+     */
     stickyColumns: PropTypes.number,
+    /**
+     * If <code>true</code> or <code>very</code>, the Data Grid will bleed off the edge.
```
Comment (cbrewer1689) [inline src/dataDisplay/dataGrid/dataGrid.jsx:79]: Backticks instead of `<code>`

**Review by cbrewer1689 — CHANGES_REQUESTED** [review]

**Comment by cbrewer1689** [comment]
@War777, please fix the conflict.

File: src/dataDisplay/dataGrid/dataGrid.jsx (hunk 1 above)
Comment (War777) [reply src/dataDisplay/dataGrid/dataGrid.jsx]: done

File: src/dataDisplay/dataGrid/dataGrid.jsx (hunk 2 above)
Comment (War777) [reply src/dataDisplay/dataGrid/dataGrid.jsx]: done

**Comment by groberts314** [comment]
PR Review Checkpoint Meeting Thurs 2020-12-10.
Cameron to re-review changes from Oscar.  If satisfied :shipit:

**Review by cbrewer1689 — APPROVED** [review]

**Pull request merged**